sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosException.java

```diff
@@ -281,6 +281,22 @@ CosmosException setDiagnostics(CosmosDiagnostics cosmosDiagnostics) {
         return this;
     }
 
+    /**
+     * Gets the request charge as request units (RU) consumed by the operation.
+     * <p>
+     * For more information about the RU and factors that can impact the effective charges please visit
+     * <a href="https://docs.microsoft.com/en-us/azure/cosmos-db/request-units">Request Units in Azure Cosmos DB</a>
+     *
+     * @return the request charge.
+     */
+    public double getRequestCharge() {
```

Comment:
This is great, thank you!
Please mention this issue in the PR as this API solves the github issue ask: https://github.com/Azure/azure-sdk-for-java/issues/13488

Reply:
Done